api/prisma/schema.prisma

```diff
@@ -36,6 +36,7 @@ model Session {
   revokedAt           DateTime?
   referenceTokenId    String
   isUserCreated       Boolean   @default(false)
+  originRestriction   String
```

Comment:
`originRestriction = '*'` does sounds to me as "all origins are restricted", therefore, the suggestion is to rename it to `allowedOrigins`